seqr/utils/elasticsearch/es_search.py

```diff
@@ -288,6 +288,10 @@ def _parse_annotation_overrides(self, annotations, pathogenicity):
         clinvar_terms, hgmd_classes = _parse_pathogenicity_filter(pathogenicity or {})
 
         annotations = {k: v for k, v in (annotations or {}).items() if v}
+        # Temporal code for migrating SV data file formats, will be delete after the migration finished
+        if annotations.get('structural_consequence') and 'MSV_EXON_OVERLAP' in annotations['structural_consequence']:
+            annotations['structural_consequence'].append('MSV_EXON_OVR')
+        # End of the temporal code
```

Comment:
theres no need for an end comment, just leave a blank line at the end to indicate what the temporary code block is